api/types/types.go

```diff
@@ -556,6 +556,21 @@ type DiskUsageOptions struct {
 	Types []DiskUsageObject
 }
 
+// ContainerUsage contains container usage part of response of Engine API:
+// GET "/system/df"
+type ContainerUsage struct {
+	ID         string `json:"Id"`
+	Image      string
+	ImageID    string
+	Command    string
+	Created    int64
```

Comment:
Looks like we already did this for regular containers, so probably should be kept, but `time.Time` would've been good for this (as for `Volumes.CreatedAt`).

We could consider adding `json:",omitempty"` to these fields though

Reply:
Let's leave that for a separate PR and focus just on "trimming down" here without changing the existing functionality too much